cmd/ddev/cmd/start.go

```diff
@@ -28,28 +30,34 @@ provide a local development environment.`,
 		dockerutil.EnsureDdevNetwork()
 	},
 	Run: func(cmd *cobra.Command, args []string) {
-		appStart()
-	},
-}
+		apps, err := getRequestedApps(args, startAll)
+		if err != nil {
+			util.Failed("Unable to start project(s): %v", err)
+		}
 
-// appStart is a convenience function to encapsulate startup functionality
-func appStart() {
-	app, err := ddevapp.GetActiveApp("")
-	if err != nil {
-		util.Failed("Failed to start project: %v", err)
-	}
+		if len(apps) == 0 {
+			output.UserOut.Printf("There are no stopped projects to start.")
+		}
 
-	output.UserOut.Printf("Starting environment for %s...", app.GetName())
+		for _, app := range apps {
+			if app.SiteStatus() == ddevapp.SiteRunning {
+				continue
+			}
 
-	err = app.Start()
-	if err != nil {
-		util.Failed("Failed to start %s: %v", app.GetName(), err)
-	}
+			output.UserOut.Printf("Starting environment for %s...", app.GetName())
```

Comment:
We could do better than this traditional error message. I've never quite understood what "environment" meant. Maybe it was from the old days with "staging" and "prod". Let's do "Starting project" or just "Starting %s".